aequilibrae/matrix/aequilibrae_matrix.py

```diff
@@ -677,6 +678,8 @@ def __flush(self, obj_to_flush: np.memmap):
 
     def __getattr__(self, mat_name: str):
         if mat_name in object.__dict__:
+            if mat_name == "matrix" and self.__omx:
+                warnings.warn("You can't access OMX matrix cores like that")
```

Comment:
Should there be an early exit here? If the use can't access the matrix like that should it still return `self.__dict__[mat_name]`? Also warnings are only raised once per source file location so repeatedly hitting this will be silent